### PR TITLE
Add cert-manager team to the cert-manager test jobs approvers list

### PR DESCRIPTION
**Pull request by wallrj**
```release-note
NONE
```

### CODE REVIEW

File: config/jobs/cert-manager/OWNERS [hunk 1]
```diff
@@ -1,5 +1,6 @@
 approvers:
 - munnerz
 - meyskens
+- wallrj
```
Comment (simplyzee) [inline config/jobs/cert-manager/OWNERS:4]: Be good to add the CM team here so that those within the team can approve changes on behalf of CM

**Review by simplyzee — APPROVED** [review]

File: config/jobs/cert-manager/OWNERS (hunk 1 above)
Comment (wallrj) [reply config/jobs/cert-manager/OWNERS:8]: I added individuals, but can I use @jetstack/team-cert-manager  instead?

Comment (simplyzee) [reply config/jobs/cert-manager/OWNERS:8]: Not sure if that's possible right now  - https://github.com/kubernetes/test-infra/issues/10065

**Pull request merged**